src/components/MainSidebar/ProgramDimensionsPanel/ProgramDimensionsList.js

```diff
@@ -24,19 +24,20 @@ const ProgramDimensionsList = ({
         <DimensionsList
             setIsListEndVisible={setIsListEndVisible}
             dimensions={dimensions}
+            isSelected={isSelected}
             error={error}
             fetching={fetching}
             loading={loading}
-            programName={programName}
+            programName={program.name}
             searchTerm={searchTerm}
         />
     )
 }
 
 ProgramDimensionsList.propTypes = {
     inputType: PropTypes.string.isRequired,
-    programId: PropTypes.string.isRequired,
-    programName: PropTypes.string.isRequired,
+    isSelected: PropTypes.func.isRequired,
```

Comment:
Was confused, thought this was a boolean, which didn't make sense. Maybe a name that suggests it is a function (I know it's not as pretty, but `getIsSelected` or similar)